src/price/TWAPMean.ts

```diff
@@ -0,0 +1,14 @@
+import {mean} from './mean';
+import type {BarPrice} from './types';
+
+/**
+ * Time-Weighted Average Price
+ * @param values bar values
+ */
+export const TWAPMean = (values: BarPrice[]): number => {
```

Comment:
I would prefer names like: `TimeWeightedMeanPrice`

Reply:
indeed, I will rename it. will start from a lowercase letter since it's a function.